src/metabase/search/config.clj

```diff
@@ -91,6 +90,15 @@
    :view-count          2
    :text                10})
 
+(def weights
+  "Strength of the various scorers. Copied from metabase.search.in-place.scoring, but allowing divergence."
+  (atom default-weights))
```

Comment:
why an atom but not a setting? our stats are deployed as a cluster with two instances so you might get different result on each request

Reply:
also, defsetting has "per-user" setting too in case you want to experiment this in isolation

Reply:
My original thinking was that this would be across all users intentionally, but perhaps it makes sense to scope per user if it resets on deploy anyway...

Didn't think about using a setting, that's an easy way to get persistence across deploys too. We're only going to use this in Stats, so clusters aren't much of a concern.

Reply:
yea, I just thought it'd be hard toats if you get b test this on stounced between servers bc stats runs on a cluster.

Reply:
Ah, I thought Stats only scaled up during HA deploys.

Incorporated the settings idea, as global with persistence seems the most useful for getting feedback from people.